db/migrate/20150107161258_add_admin_to_users.rb

```diff
@@ -0,0 +1,5 @@
+class AddAdminToUsers < ActiveRecord::Migration
+  def change
+    add_column :users, :admin, :boolean, default: false
```

Comment:
We considered this knowing that this would allow our app to scale, but since we would not implement more roles, we added the admin column instead